infra/db.bicep

```diff
@@ -41,25 +61,30 @@ resource postgresConfig 'Microsoft.DBforPostgreSQL/flexibleServers/configuration
   }
 }
 
-resource sqlDeploymentScript 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
-  name: '${name}-deployment-script'
+resource sqlDeploymentScriptSetup 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
```

Comment:
Looking at this now, I think we should move this into a separate file and treat it as a separate module in Bicep, rather than have it within the server setup. This should address the issue later on where we can't use the outputs of this module in the admin/proxy module, because their SP's aren't setup.